client/components/layout/main-layout/header/Header.tsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import TopNavigationBar from './TopNavigationBar';
+import LinktaMainLogo from './LinktaMainLogo';
 
 const Header = () => {
-  return <div>Header Placeholder</div>;
+  return (
+    <>
+      <LinktaMainLogo />
```

Comment:
I would put this inside the `<nav />` element

Reply:
Thanks for the inspiration I think I will place it and the nav bar together in the header tag for conditional rendering of nav bar tabs:)